src/datasources/tag/components/TagQueryEditor.tsx

```diff
@@ -51,19 +50,21 @@ export function TagQueryEditor({ query, onChange, onRunQuery, datasource }: Prop
           value={query.workspace}
         />
       </InlineField>
-      <InlineField label="Properties" labelWidth={14} tooltip={tooltips.properties}>
-        <InlineSwitch onChange={onPropertiesChange} value={query.properties} />
-      </InlineField>
+      {query.type === TagQueryType.Current && (
+        <InlineField label="Properties" labelWidth={14} tooltip={tooltips.properties}>
+          <InlineSwitch onChange={onPropertiesChange} value={query.properties}/>
+        </InlineField>
+      )}
     </>
   );
 }
 
 const tooltips = {
-  queryType: `Current allows you to visualize the most recent tag value. History allows you to
-              visualize tag values over time. Historical values use the time range set on the
+  queryType: `Current allows you to visualize the most recent value of multiple tags. History allows you to
+              visualize the values of multiple tags over time. Historical values use the time range set on the
               dashboard and are decimated according to the width of the panel.`,
 
-  tagPath: `The full path of the tag to visualize. You can enter a variable into this field.`,
+  tagPath: `The path to search for the tags you want to visualize. You can enter a variable into this field and use glob-style patterns as wildcards.`,
```

Comment:
@vpstoynova Could you review these strings? My personal opinion:

"Current allows you to visualize the most recent value of **one or more tags**"

"The path to search for the tags you want to visualize. **Use * as a wildcard to match multiple tags.**"

Reply:
I think your suggestions look good.

Reply:
@TigranVardanyan it doesn't look like this feedback was ever incorporated

Reply:
@mure updated